switchmap/server/db/db.py

```diff
@@ -285,3 +288,49 @@ def db_add_all(error_code, instances, die=True):
 
     # Return
     return result
+
+
+def db_bulk_insert(error_code, model, mappings, die=True):
```

Comment:
We need to streamline this.

1. Rename to `db_insert_row`
2. All table `*.insert_row` functions must refer to `db.db_insert_row` instead of `db.db_add_all` 
3. Make all insertions use this function. 
4. Remove all references to `db_add_all` from the code base. It's outdated as you mentioned.